pkgs/games/dwarf-fortress/default.nix

```diff
@@ -35,6 +37,22 @@ let
   dfHackWorksWithCurrentVersion = true;
   dfHackEnabled = dfHackWorksWithCurrentVersion && enableDFHack;
 
+  themesThatWorkWithCurrentVersion = [
+    "phoebus"
+  ];
+  themeEnabled = builtins.elem theme themesThatWorkWithCurrentVersion;
+  themeToUse = if themeEnabled then theme else null;
+  themePkg = if (themeToUse != null)
+    then
+      import (./. + ("/themes/" + themeToUse + ".nix")) {
```

Comment:
Why do you hardcode the available themes? I think it's much simpler to just accept a theme directory as `theme` parameter and integrate that, instead of having to package each theme in a prettty cumbersome way.

(Simplified: `mv ${theme} $out/` - just copy anything from the supplied theme into the DF installation)